app_dart/lib/src/service/config.dart

```diff
@@ -122,12 +122,10 @@ class Config {
   }
 
   Future<Uint8List> _getValueFromDatastore(String id) async {
-    final CocoonConfig cocoonConfig = CocoonConfig()
-      ..id = id
-      ..parentKey = _db.emptyKey;
-    final CocoonConfig result = await _db.lookupValue<CocoonConfig>(cocoonConfig.key);
-
-    return Uint8List.fromList(result.value.codeUnits);
+    final CocoonConfig? result = await _db.lookupOrNull<CocoonConfig>(
```

Comment:
Is this updated file a refactor or is it related to the local server somehow?

Reply:
It's a refactor to work better on the local server. Otherwise, we get null safety runtime errors for a lot of unused values.